akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraProjectionImpl.scala

```diff
@@ -159,7 +159,7 @@ import akka.stream.scaladsl.Source
    * This is mainly intended to be used by the TestKit allowing it to attach a TestSink to it.
    */
   @InternalApi
-  override private[projection] def mappedSource()(implicit system: ActorSystem[_]): Source[Done, _] = {
+  override private[projection] def mappedSource()(implicit system: ActorSystem[_]): Source[Done, Future[Done]] = {
```

Comment:
As discussed [here](https://github.com/akka/akka-projection/pull/250#discussion_r442285497), the Mat type needs to fixed. At the end, it's not `NotUsed` but `Future[Done]`. That's because of the last transformation we do in `InternalProjectionState.mappedSource`.